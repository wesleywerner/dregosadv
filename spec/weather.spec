; sky.enabled = true
superbrief
gonear clearing portal
go jungle path
examine sky
listen
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
examine sky
listen
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
z
look up
listen
