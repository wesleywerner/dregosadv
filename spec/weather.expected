  THIS GAME IS IN DEBUG MODE
  
  Dr Ego and the Egg of Man-Toomba
  An Interactive Fiction Story
  by Special Agent.
  Release 1 / Serial number 400410 / Inform v6.33 Library v6.12.3pre SX
  
  The Maro River (in the canoe)
. 
  In the canoe you can also see a trap (which is closed and empty), a fishing
  spear and a fishing net.
  
> >  ; == 1
  
> >  Dr Ego and the Egg of Man-Toomba is now in its "superbrief" mode, which
  always gives short descriptions of locations (even if you haven't been
  there before).
  
> >  Clearing
  
  A hut is due south.
  A jungle path is due east.
> >  Jungle Path
  
  A clearing in the jungle is due west.
  The river shore is due south.
> >  (the sky)
  Rolling clouds move overhead.
> >  You hear the sounds of the jungle - faint rustling of leaves in the wind,
  faint chirps of insects, and occasionally some small critter moving under
  cover of the dense foliage.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  It starts to rain.
  
> >  Time passes.
  Cold drops fall on your back.
  
> >  Time passes.
  
> >  Time passes.
  Cold drops land on your head.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  Cold drops land on your face.
  
> >  Time passes.
  
> >  Time passes.
  Cold drops land on your arms.
  
> >  Time passes.
  
> >  Time passes.
  Rain drops fall on your face.
  
> >  (the sky)
  You feel the rain on your face as you look up. You see dark rain bearing
  clouds moving overhead. Drops of rain land on your face.
  
> >  You hear the pitter-patter of rain around you.
  
> >  Time passes.
  
> >  Time passes.
  Cold drops fall on your head.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  Drops of rain land on your back.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  Lightning arcs across the sky.
  
> >  Time passes.
  
> >  Time passes.
  The air trembles with the rolling thunder.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  Rain drops spatter on your face.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  Drops of rain fall on your face.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  Lightning flashes in the distance.
  
> >  Time passes.
  The sound of thunder rumbles overhead.
  
> >  Time passes.
  
> >  Time passes.
  The rain eases off.
  
> >  Time passes.
  Clouds flicker menacingly with lightning.
  
> >  Rolling clouds move overhead.
> >  You hear the sounds of the jungle - faint rustling of leaves in the wind,
  faint chirps of insects, and occasionally some small critter moving under
  cover of the dense foliage.
  
> >